lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +210,37 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.urls = []
+        self._config = config
+        self._seed_urls = config.get_seed_urls()
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        url = article_bs['href']
+        if isinstance(url, str):
+            return url
+        return url[0]
 
     def find_articles(self) -> None:
         """
         Finds articles
         """
-        pass
+        for seed in self._seed_urls:
+            response = make_request(seed, self._config)
```

Comment:
What if request happens to be unsuccessul?

Reply:
done